packages/common/src/operations.graphql

```diff
@@ -11,84 +11,38 @@ query itemsWithStockLines(
     sort: { key: $key, desc: $desc }
     filter: $filter
   ) {
-    ... on ConnectorError {
```

Comment:
Updating the common operations. Main things here are that there is no longer a `ConnectorError` for queries which return array of things. We're still getting an `XResponse` object returned - i.e. `ItemsResponse` for `items` query, but that union has only one member.

That means we definitely get the `ItemConnector` object back, so we don't need to use a guard. However, it means we still keep the `... on ItemConnector` syntax..

So instead of just doing

```
query items {
    totalCount
    nodes { ...}
}
```

```
query items {
    ... on ItemConnector { 
        totalCount
        nodes { ...}
    }
}
```

But effectively what we get returned is the same, so we don't need to worry about checking what type it is anymore (at the moment?)